app/assets/stylesheets/application.scss

```diff
@@ -42,3 +42,22 @@
 .home-form {
   color: #ffffff;
 }
+
+.entries {
+  background-color: #ffffff;
+  border: 1px solid #999999;
+  list-style: none;
+  margin: 15px;
+  text-align: center;
+}
+
+body {
+  background-color: #e0e0e5;
+}
+
+.stats {
+  background-color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.